src/MediaFrameListenerBridge.cpp

```diff
@@ -168,7 +168,9 @@ void MediaFrameListenerBridge::RemoveListener(RTPIncomingMediaStream::Listener*
 
 void MediaFrameListenerBridge::onMediaFrame(DWORD ignored, const MediaFrame& frame)
 {
-	timeService.Async([=, frame = std::shared_ptr<MediaFrame>(frame.Clone())] (auto now){
+	std::shared_ptr<MediaFrame> cloned(frame.Clone());
+	cloned->SetSSRC(ssrc);
+	timeService.Async([=, frame = std::move(cloned)] (auto now){
```

Comment:
I see,  maybe it is better to refector a bit to avoid the impresion that it is a problem in the `Clone()` function but that we need to override the ssrc of the incoming ssrc to the oitgoing ssrc
```suggestion
	timeService.Async([=, frame = std::shared_ptr<MediaFrame>(frame.Clone())] (auto now){
		//Update outgoing ssrc before multiplexing
		frame->SetSSRC(ssrc);
```